### PR TITLE
MSS-1345: Fix jackson-databind vulnerability

### PR DESCRIPTION
**Pull request by frankie297**
Force jackson-databind version to fix some vulnerability issues:
https://app.snyk.io/vuln/SNYK-JAVA-COMFASTERXMLJACKSONCORE-174736

NB: remove after upgrading play app to 2.7 (if we should ever do this)

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -7,13 +7,17 @@ import scala.collection.immutable
 val testAndCompileDependencies: String = "test->test;compile->compile"
 val awsVersion: String = "1.11.375"
 val simpleConfigurationVersion: String = "1.4.3"
+val jacksonData: String = "2.9.10.1"
+val jacksonDatatype: String = "2.9.8"
```
Comment (jacobwinch) [inline build.sbt]: Is `jacksonDatatype` used?

File: build.sbt [hunk 2]
```diff
@@ -7,13 +7,17 @@ import scala.collection.immutable
 val testAndCompileDependencies: String = "test->test;compile->compile"
 val awsVersion: String = "1.11.375"
 val simpleConfigurationVersion: String = "1.4.3"
+val jacksonData: String = "2.9.10.1"
```
Comment (jacobwinch) [inline build.sbt]: It might be worth adding a comment here to explain why we're pinning the `jackson-databind` version?

File: build.sbt (hunk 1 above)
Comment (frankie297) [reply build.sbt]: good point, i don't think it is needed. Thanks for spotting (i might have just carried this over from another repo as I have done the same change multiple times) 👍

**Comment by frankie297** [comment]
@jacobwinch Just so you are aware I have actually upgraded to a newest version due to a recent vulnerability in `2.9.10.1`

File: build.sbt (hunk 2 above)
Comment (frankie297) [reply build.sbt]: 👍

**Pull request merged**